client/components/mma/shared/CardDisplay.tsx

```diff
@@ -156,6 +156,7 @@ export const CardDisplay = (props: CardDisplayProps) => {
 				`}
 			>
 				ending {props.last4}
+				data-qm-masking="blocklist"
```

Comment:
This should be on the `span` element rather than inside so it doesn't get shown after the card details.